src/components/accessories/table/Table.tsx

```diff
@@ -23,19 +23,22 @@ import {
 import "./styles.scss";
 import TableBodyRow from "./TableBodyRow";
 import { IProps, TActions } from "./types";
-import { defaultComparator } from "../../../libraries/sortUtils/sortUtils";
+import {
+  dateComparator,
+  defaultComparator,
+} from "../../../libraries/sortUtils/sortUtils";
 import ConfirmationDialog from "../confirmationDialog/ConfirmationDialog";
 import { useTranslation } from "react-i18next";
 import warningIcon from "../../../assets/warning-icon.png";
 
 const Table: FunctionComponent<IProps> = ({
   rowData,
   tableHeader,
+  dateFields = [],
   labelData,
```

Comment:
But I was thinking if it is not possible to factorize all these property with just one array of objects like:
`fields= [ {name: xxxxx, header: true|false, date: true|false, order: 1/2/3... }, ...]` 

And now in the table component we just use the same fields array. We can use map function to extract all what we want. What you have done is ok too. What do you think about my suggestion?

Reply:
This is a great suggestion in order to make the code more simple and maintenaible. I'm gonna open a new Jira task.

Reply:
Ok you are right. It should be a new task.